main.py

```
@@ -1,34 +1,20 @@
-    font = pygame.font.Font(None, 50)  # Default font, size 74
-    data_text = font.render(f"{gui.data}", True, (0,0,0))
-    wdata_text = font.render(f"{gui.wdata}", True, (0,0,0))
-    data1_text = font.render(f"{gui.mousbuttondown}", True, (0,0,0))
-    screen.blit(data_text, (0, 0))
-    screen.blit(data1_text,(0, data_text.get_height()+5))
-    screen.blit(wdata_text,(0, data_text.get_height()+5 + data1_text.get_height() + 5))
-    return window
-
-
 def load_gui():
     os = MOS()
```

Ruff (0.13.1)

Local variable `os` is assigned to but never used

Remove assignment to unused variable `os`

(F841)